source/a_box/main.c

```diff
@@ -288,14 +293,13 @@ void heartBeatLED()
 void monitorStatus()
 {
    uint8_t bms_err, imd_err;
-   bms_err = orionErrors();
+   bms_err = orionErrors() | readTemps(&tmu);
```

Comment:
Update line 63 of orion.c to remove setting the temperature fault since you are moving this stuff to tmu.c

Reply:
done